config/config.json

```diff
@@ -42,6 +42,18 @@
     "metrics_filename": "./cproc_metrics.json"
   },
   "plotting": {
+    "dstats": {
+      "out_prefix" : "output",
+      "aggregate" : "True",
+      "to_plot" : {
+        "Network": "True",
+        "ColPanel" : ["CPUPERC", "NetRX"],
+        "ValueCluster" : [],
+        "DegColPanel" : ["VmPeak"],
+        "SettlingTime" : "True",
+        "Compare" : "True"
+      }
+    },
```

Comment:
Hmmm, maybe we didn't understood this correctly. The idea of this configuration was that it should only affect those metrics in prometheus. Neither jordi's config or mine regarding on our own collected metrics are here...

I guess we should discuss this in our next meeting.

Reply:
Both `cadvisor` and `container-proc` are focused on producing only one analysis/plot: the combined plots for comparison purposes. But  `host-proc`  and `dstats` have handful of additional plots, and these can be selected via config.json now. Also, `host-proc` can track additional parameters, like Data Segment usage, Stack Segement usage etc; and network measurements from different places. So it is useful to select those params/analysis we would want to perform, hence the addition of `host-proc`/`dstats` keys to the plots. as the analysis module grows, we could to add the same for `cadvisor`/`container-proc` as well.